ts-rs/src/lib.rs

```diff
@@ -727,3 +727,22 @@ impl_primitives! {
 }
 #[rustfmt::skip]
 pub(crate) use impl_primitives;
+
+
+#[doc(hidden)]
+#[derive(Copy, Clone, Debug, Hash, Eq, PartialEq, Ord, PartialOrd)]
+pub struct Dummy;
+
+#[doc(hidden)]
```

Comment:
Does `#[doc(hidden)]` do anything on the `impl`s?

Reply:
I have no idea 😆. Just didn't wanna risk it

Reply:
I thought it might because docs.rs shows what traits a type implements, but since the type itself has `#[doc(hidden)]` it may be redundant